### PR TITLE
fix(FEC-9379): add detection for chromium Edge

### PR DESCRIPTION
**Pull request by Yuvalke**
### Description of the Changes

add user detection for dev or canary Chromium Edge

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/utils/env.js [hunk 1]
```diff
@@ -20,7 +20,11 @@ const DeviceParser = [
   [[UAParser.DEVICE.TYPE, UAParser.DEVICE.SMARTTV]]
 ];
 
-let Env = new UAParser(undefined, {browser: SAMSUNGBrowserParser, device: DeviceParser, os: LGOSParser}).getResult();
+const EdgeChromium = [[/(edg)\/((\d+)?[\w.]+)/i], [[UAParser.BROWSER.NAME, 'Edge'], UAParser.BROWSER.VERSION, UAParser.BROWSER.MAJOR]];
+
+const browserDetect = EdgeChromium.concat(SAMSUNGBrowserParser);
```
Comment (RoyBregman) [inline src/utils/env.js]: is browserDetect the best naming for this variable? maybe something like samsungOrChromiumDetect?

Comment (Yuvalke) [reply src/utils/env.js]: I don't think so, according to your suggestion next time it'll be samsungOrChromiumSomethingElseDetect and after that.. bottom line you don't want to create infinite variable names which will be meaningless,  I used the the name of the attribute that I want to parse.

Comment (RoyBregman) [reply src/utils/env.js]: ok, so maybe just uppercase cause its a constant

Comment (Yuvalke) [reply src/utils/env.js]: About the constant, it's `Edge` cause in chromium it wrote `Edg` in user agent and in other cases is `Edge`.

Comment (RoyBregman) [reply src/utils/env.js]: my bad, I am used that constants are uppercased. But not in Kaltura :)

**Pull request merged**